packages/cairo-dock-core/build.sh

```diff
@@ -19,3 +20,19 @@ fi
 
 TERMUX_CMAKE_BUILD="Unix Makefiles"
 
+termux_step_post_make_install(){
+	if [ $TERMUX_ARCH == "arm" ]; then
+		chmod +x ${TERMUX_PREFIX}/lib32/libgldi.so
+		chmod +x ${TERMUX_PREFIX}/lib32/cairo-dock/*
+		echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:$TERMUX_PREFIX/lib32" > $TERMUX_PREFIX/etc/profile.d/cairo-dock.sh
+	elif [ $TERMUX_ARCH == "i686" ]; then
+		chmod +x ${TERMUX_PREFIX}/lib32/libgldi.so
+                chmod +x ${TERMUX_PREFIX}/lib32/cairo-dock/*
+		echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:$TERMUX_PREFIX/lib32" > $TERMUX_PREFIX/etc/profile.d/cairo-dock.sh
+	else
+		chmod +x ${TERMUX_PREFIX}/lib64/libgldi.so
+                chmod +x ${TERMUX_PREFIX}/lib64/cairo-dock/*
+		echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:$TERMUX_PREFIX/lib64" > $TERMUX_PREFIX/etc/profile.d/cairo-dock.sh
+	fi
+
```

Comment:
Please get rid of multilib sets. Termux doesn't support this. We also do not recommend using `LD_LIBRARY_PATH` anymore - it breaks stuff that uses Android libraries such as OpenSLES. All libraries **must** be installed into `$TERMUX_PREFIX/lib`. And don't set executable bit on libraries, especially from postinst script.

Termux file system layour: https://github.com/termux/termux-packages/wiki/Termux-file-system-layout